### ISSUE TITLE
Some bug reported by terminal, not sure what it is myself

### ISSUE THREAD

**Issue by MOHSENUL** [PowerShell/PSReadLine#4160]
### Prerequisites

- [X] Write a descriptive title.
- [X] Make sure you are able to repro it on the [latest released version](https://www.powershellgallery.com/packages/PSReadLine)
- [X] Search the existing issues, especially the pinned issues.

### Exception report

```console
Oops, something went wrong.  Please report this bug with the details below.
Report on GitHub: https://github.com/lzybkr/PSReadLine/issues/new
```

### Screenshot

![image](https://github.com/user-attachments/assets/8829a868-506e-4151-8614-e962b1ad0e32)

### Environment data

```console
PS Version: 5.1.19041.4842
PS HostName: ConsoleHost
PSReadLine Version: 2.0.0-beta2
PSReadLine EditMode: Windows
OS: 10.0.19041.1 (WinBuild.160101.0800)
BufferWidth: 120
BufferHeight: 3
```

### Steps to reproduce

basically killing the terminal and running it again produces it (im kinda new to coding if my descriptions arent good enough im sorry)

### Expected behavior

just execute the code as typed out in the screenshot 

### Actual behavior

terminal reports a bug
